app/Http/Controllers/UsersController.php

```diff
@@ -347,6 +312,48 @@ public function postEdit(UpdateUserRequest $request, $id = null)
     }
 
     /**
+    * Maps Request Information to a User object
+    *
+    * @auther [Daniel Meltzer] [<parallelgrapefruit@gmail.com>]
+    * @since [v3.0]
+    * @param User $user
+    * @param Request $request
+    * @return User
+    */
+    private function extractUserDataFromRequest(User $user, Request $request)
```

Comment:
It just feels weird to have a class method that does literally nothing other than the stuff that controllers should be doing anyway. :-/